mastodon/src/main/java/org/joinmastodon/android/ui/displayitems/StatusDisplayItem.java

```diff
@@ -233,20 +230,25 @@ public static ArrayList<StatusDisplayItem> buildItems(BaseStatusListFragment<?>
 
 		LegacyFilter applyingFilter=null;
 		if(status.filtered!=null){
-			for(FilterResult filter:status.filtered){
+			List<FilterResult> filters = status.filtered;
+
+			//add a client side filter to filter posts that have no alt text
+			//it only applies when activated in the settings
+			AltTextFilter altTextFilter=new AltTextFilter(FilterAction.WARN, EnumSet.allOf(FilterContext.class));
```

Comment:
You shouldn't put the client filters declarations all here, I would like a separate method to return them all, and we could then parse it on here. I want it to be this way so that we can add separate filters without messing too much with the the StatusDisplayItem buildItems method